GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -61,7 +62,8 @@ public sealed partial class FormBrowse : GitModuleForm, IBrowseRepo
         private readonly TranslationString _configureWorkingDirMenu = new TranslationString("Configure this menu");
 
         private readonly TranslationString _directoryIsNotAValidRepositoryCaption = new TranslationString("Open");
-        private readonly TranslationString _directoryIsNotAValidRepository = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to remove it from the recent repositories list?");
+        private readonly TranslationString _directoryIsNotAValidRepositoryMainInstruction = new TranslationString("The selected item is not a valid git repository.");
+        private readonly TranslationString _directoryIsNotAValidRepositoryCommandButtons = new TranslationString("Remove the selected invalid repository{0:{2}Remove all {1} invalid repositories|}");
```

Comment:
Please create two different strings, it will make it easier to maintain and for the translators to work with.
You could also then use `string.Format` instead of `Smart.Format`